heron/executor/src/python/heron_executor.py

```diff
@@ -342,6 +343,8 @@ def init_from_parsed_args(self, parsed_args):
     self.health_manager_mode = parsed_args.health_manager_mode
     self.health_manager_classpath = '%s:%s'\
         % (self.scheduler_classpath, parsed_args.health_manager_classpath)
+    self.enable_verbose_gc_log = parsed_args.enable_verbose_gc_log \
```

Comment:
You need to use boolean instead of string.

The string 'false' is true in a if statement.

Reply:
Thank you. Does that look better?